src/Domain/Charity.php

```diff
@@ -148,4 +162,44 @@ public function setRegulatorNumber(?string $regulatorNumber): void
     {
         $this->regulatorNumber = $regulatorNumber;
     }
+
+    public function setTbgApprovedToClaimGiftAid(bool $tbgApprovedToClaimGiftAid): void
+    {
+        $this->tbgApprovedToClaimGiftAid = $tbgApprovedToClaimGiftAid;
+    }
+
+    public function updateFromSfPull(
+        string   $charityName,
+        ?string  $stripeAccountId,
+        ?string  $hmrcReferenceNumber,
+        ?string  $giftAidOnboardingStatus,
+        ?string  $regulator,
+        ?string  $regulatorNumber,
+        DateTime $time,
+    ): void
+    {
+        $this->setName($charityName);
+        $this->setStripeAccountId($stripeAccountId);
+
```

Comment:
maybe worth adding in a check that `$giftAidOnboardingStatus` is a status we've heard of and throwing if not - either with `in_array` and throw or by calling `Assertion::inArray(mixed $value, array $choices);` from https://github.com/beberlei/assert